src/build.d

```diff
@@ -247,7 +233,7 @@ version(Windows)
         Dependency dependency = {
             target: env["G"].buildPath(targetName).exeName,
             sources: [`vcbuild\` ~ targetName],
-            name: "(DC) MSCV-LIB " ~ targetName,
+            name: "(DC) MSCV-LIB",
```

Comment:
Maybe:

```suggestion
            name: "(DC) MSCV-LIB " " ~ targetName.baseName,
```

Reply:
`targetName` is already the `baseName`, hence the need for `buildPath` for `target`.  Am I missing something?

Reply:
Maybe `targetName.exeName`?

Reply:
Yeah whatever you prefer. All is more information than just MSCV-LIB